documentation/modules/con-creating-managing-connectors.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-connect.adoc
+
+[id='con-creating-managing-connectors-{context}']
+
+= Creating and managing connectors
+
+When you have created a container image for your connector plug-in, you need to create a connector instance in your Kafka Connect cluster. You can then configure, monitor, and manage a running connector instance. For example, you can:
+
+* Check the status of a connector instance
+* Increase or decrease the number of tasks
+* Restart failed tasks
+* Pause a connector instance
+* Delete a connector instance
+
+{ProductName} provides two different APIs for creating and managing connectors:
+
+* `KafkaConnector` resources (referred to as `KafkaConnectors`)
+* Kafka Connect REST API
+
+NOTE: Currently, `KafkaConnectors` do not support restarting failed tasks or pausing and resuming connectors. You need to use the Kafka Connect REST API to perform those tasks.
```

Comment:
Support for pause/resume has been merged...

```suggestion
NOTE: Currently, `KafkaConnectors` do not support restarting failed tasks. You need to use the Kafka Connect REST API to to this.
```

Reply:
OK, that's updated.